README.md

```diff
@@ -2,10 +2,19 @@
 
 This is the Repository for the codebase concerning the [Zcash RFP](https://forum.zcashcommunity.com/t/rfp-zcash-uniffi-library/41335) published some time ago on the Forum. [Eiger](https://www.eiger.co/) won the grant to realize the RFP with [this proposal](https://forum.zcashcommunity.com/t/zcash-uniffi-library-rfp/43468).
 
+⚠️ Production use its discouraged, as this project its still under heavy development.
```

Comment:
*its* to *is*

Reply:
Fixed at 9143c63568e1b3ca3f42a03361ad1d635450efb6